### ISSUE TITLE
Experience on hyper-parameters

### ISSUE THREAD

**Issue by Huage001** [GeorgeCazenavette/mtt-distillation#14]
Dear author,

Thank you for your great solution on dataset distillation! Recently I am working on my own datasets but find that the performance is somewhat sensitive to the hyper parameters. Could you please provide some insights on how to choose the hyper-parameters like syn_steps, expert_epochs, max_start_epoch, learning rate, etc? Thanks in advance!

**Comment by GeorgeCazenavette** [comment]
You can find the hyper-parameters we used for our experiments here: https://github.com/GeorgeCazenavette/mtt-distillation/issues/7

In general, syn_steps was just however much we could fit on our GPUs, expert_epochs was just 2, and max_start_epoch was dependent on how the training of the experts went. You can generally start with max_start_epoch=1 and then try higher numbers until the training gets too unstable (because the expert updates are too small).
